src/docs/reference/core/Configuration/README.md

```diff
@@ -54,6 +59,22 @@ In addition you can specify a `Tenant` setting by using the Tenant Name, in this
 }
 ```
 
+You can also configure options from code in the web project's `Startup` class as explained in the [ASP.NET documentation](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options). A lot of Orchard features have options support while they don't offer default static configuration behavior so for those you need to write your own configuration code.
```

Comment:
ok cool, can you break this and the next sentence up more (need line breaks for the markdown), use `IOptions`, `Orchard Core`, and explain that if you do this, the values in the admin site settings will not be correct, or used, though they will still be visible

Reply:
I'm not sure I understand what you mean by breaking it up and line breaks. These are short paragraphs already:
![image](https://user-images.githubusercontent.com/1976647/80810677-a36e7f00-8bc4-11ea-86f6-f503a4ab2732.png)

I don't understand either what you mean by "use IOptions, Orchard Core".

I think your other suggestion adds redundancy as it's already mentioned the settings will be overridden, and we don't want to explain the overall usage of options instead of the person looking at the official documentation which is linked there. But I've added more explanation.